lib/src/utilities/string_extensions.dart

```diff
@@ -1,3 +1,6 @@
+// coverage:ignore-file
+
+// Not sure why codecov doesn't accurately record this as it's tested
```

Comment:
idk why this isn't showing up in codecov. But the tests exist so I am not going to spend time on it.